Raulin0/backup-juggler
# feat: add `bj` entry point

## Description
This pull request updates the Pyproject.toml file and the test_cli.py file. It refactors the Pyproject.toml file by updating the dependencies and adding a new script entry for the 'bj' command. It also improves the organization and readability of the code in the test_cli.py file.

## Changes Made

- Added a new script entry for the 'bj' command in the Pyproject.toml file.
- Added a new test case to ensure the 'bj' script runs successfully using Poetry.

## Motivation and Context
The purpose of these changes is to ensure the project's dependencies are up-to-date and compatible. It also improves the structure and readability of the test_cli.py file, making it easier to maintain and understand.

## How to Test

1. Verify that Poetry is installed.
2. Run 'poetry install' to install the project's dependencies.
3. Execute the existing tests in the test_cli.py file and ensure they pass successfully.
4. Run the 'bj' command using Poetry and verify that it executes without any errors.
5. Check that the project functions as expected after the updates.

## Checklist
